docs/snippets/common/chromatic-github-action.js.mdx

```diff
@@ -15,7 +15,13 @@ jobs:
     # Job steps
     steps:
       - uses: actions/checkout@v3
-      - run: yarn
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: 'npm'
```

Comment:
@natehouk when you're able, can you revert this to include `yarn`? Also, if you're going to reference a node version, could you instead point it to the stable release, as not all users are already on version 20, which could introduce some unwanted issues?

Reply:
@jonniebigodes I don't think I was able to get it to work with yarn, I think that is why I switched it back to npm.

Regardless, I made the two changes you requested:
* Pinned to node version 18
* Reverted to use yarn

Please check and merge.